pkgs/by-name/cl/cliphist/package.nix

```
@@ -2,25 +2,30 @@
   lib,
   buildGoModule,
   fetchFromGitHub,
+  nix-update-script,
 }:
 
 buildGoModule rec {
   pname = "cliphist";
-  version = "0.5.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "sentriz";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-U78G7X9x3GQg3qcBINni8jWa0wSXQu+TjYChuRPPcLE=";
+    sha256 = "sha256-AWLcHUwFphfUt6gCal+/OqfRmXs7I1m2Xcshe7kPFxQ=";
```

Since you are adding an updateScript, I think some changes can also be applied to the derivation itself.

```suggestion
	  repo = "cliphist";
	  rev = "refs/tags/v${version}";
	  hash = "sha256-AWLcHUwFphfUt6gCal+/OqfRmXs7I1m2Xcshe7kPFxQ=";
```

---

Sorry I missed that. I'll update when I get home.